adobe_vipm/adobe/utils.py

```diff
@@ -0,0 +1,12 @@
+def get_actual_sku(items, sku):
+    item = next(filter(lambda item: item["offerId"].startswith(sku), items), {})
```

Comment:
`find_first`?

Reply:
I will do this refactoring in the next PR. now find_first is located in utils.py of flows package. I don't want to import it in adobe package since my rule is flows use adobe package but not the contrary.

Reply:
Fixed